crates/air_r_formatter/src/r/auxiliary/call_arguments.rs

```diff
@@ -919,6 +932,7 @@ impl Format<RFormatContext> for FormatGroupedArgument {
 }
 
 struct FormatAllArgsBrokenOut<'a> {
+    call: &'a RCallLikeArguments,
```

Comment:
The whole call is now part of this struct. This way we can peek at anything we want if needed.